argo_rollouts/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <!-- towncrier release notes start -->
 
+## 1.0.1 / 2024-05-31
+
+***Fixed***:
+
+* Update the description for the tls_ca_cert config option to use openssl rehash instead of c_rehash ([#17476](https://github.com/DataDog/integrations-core/pull/17476))
```

Comment:
```suggestion
* Update the description for the `tls_ca_cert` config option to use `openssl rehash` instead of `c_rehash` ([#17476](https://github.com/DataDog/integrations-core/pull/17476))
```